mpas_analysis/sea_ice/timeseries.py

```diff
@@ -310,45 +312,79 @@ def seaice_timeseries(config, streamMap=None, variableMap=None):
             if (compareWithObservations or
                     preprocessedReferenceRunName != 'None'):
                 # separate plots for nothern and southern hemispheres
-                timeseries_analysis_plot(config, varsNH, movingAveragePoints,
-                                         titleNH,
-                                         xLabel, units, figureNameNH,
-                                         lineStyles=lineStyles,
-                                         lineWidths=lineWidths,
-                                         titleFontSize=titleFontSize,
-                                         calendar=calendar)
-                timeseries_analysis_plot(config, varsSH, movingAveragePoints,
-                                         titleSH,
-                                         xLabel, units, figureNameSH,
-                                         lineStyles=lineStyles,
-                                         lineWidths=lineWidths,
-                                         titleFontSize=titleFontSize,
-                                         calendar=calendar)
+                if (polarPlot):
+                    timeseries_analysis_plot_polar(config, varsNH, movingAveragePoints,
+                                                   titleNH,
+                                                   figureNameNH,
+                                                   lineStyles=lineStyles,
+                                                   lineWidths=lineWidths,
+                                                   titleFontSize=titleFontSize,
+                                                   calendar=calendar)
+                    timeseries_analysis_plot_polar(config, varsSH, movingAveragePoints,
+                                                   titleSH,
+                                                   figureNameSH,
+                                                   lineStyles=lineStyles,
+                                                   lineWidths=lineWidths,
+                                                   titleFontSize=titleFontSize,
+                                                   calendar=calendar)
+                else:
```

Comment:
I would probably have flags for both to allow each to be turned on and off separately, just in case a third type of plot is added sometime in the future.